php-checks/src/main/java/org/sonar/php/checks/FormattingStandardCheck.java

```diff
@@ -205,44 +203,42 @@
     type = "BOOLEAN")
   public boolean isExtendsAndImplementsLine = true;
 
-  @Override
-  public void init() {
-    subscribeTo(
-      PHPGrammar.NAMESPACE_STATEMENT,
-      PHPGrammar.PARAMETER_LIST,
-      PHPGrammar.FUNCTION_CALL_PARAMETER_LIST,
-      PHPGrammar.FUNCTION_EXPRESSION,
-      PHPGrammar.USE_STATEMENT);
-    subscribeTo(CLASS_AND_FUNCTION);
-    subscribeTo(CONTROL_STRUCTURE);
-    subscribeTo(PHPPunctuator.RPARENTHESIS);
-  }
+//  @Override
+//  public void init() {
+//    subscribeTo(
+//      PHPGrammar.NAMESPACE_STATEMENT,
+//      PHPGrammar.PARAMETER_LIST,
+//      PHPGrammar.FUNCTION_CALL_PARAMETER_LIST,
+//      PHPGrammar.FUNCTION_EXPRESSION,
+//      PHPGrammar.USE_STATEMENT);
+//    subscribeTo(CLASS_AND_FUNCTION);
+//    subscribeTo(CONTROL_STRUCTURE);
+//    subscribeTo(PHPPunctuator.RPARENTHESIS);
+//  }
 
   @Override
-  public void visitNode(AstNode astNode) {
-    namespaceAndUseStatementCheck.visitNode(this, astNode);
-    curlyBraceCheck.visitNode(this, astNode);
-    punctuatorSpacingCheck.visitNode(this, astNode);
-    functionSpacingCheck.visitNode(this, astNode);
-    controlStructureSpacingCheck.visitNode(this, astNode);
-    indentationCheck.visitNode(this, astNode);
-    extendsImplementsLineCheck.visitNode(this, astNode);
+  public void visitScript(ScriptTree tree) {
+    namespaceAndUseStatementCheck.visitScript(this, tree);
+    curlyBraceCheck.visitScript(this, tree);
+    punctuatorSpacingCheck.visitScript(this, tree);
   }
 
-  @Override
-  public void leaveFile(@Nullable AstNode astNode) {
-    namespaceAndUseStatementCheck.leaveFile();
-  }
+//  public void visitNode(AstNode astNode) {
+//    functionSpacingCheck.visitNode(this, astNode);
+//    controlStructureSpacingCheck.visitNode(this, astNode);
+//    indentationCheck.visitNode(this, astNode);
+//    extendsImplementsLineCheck.visitNode(this, astNode);
+//  }
 
-  public void reportIssue(String msg, AstNode node) {
-    getContext().createLineViolation(this, msg, node);
+  public void reportIssue(String msg, Tree tree) {
+    context().newIssue(KEY, msg).tree(tree);
   }
 
-  public static GrammarRuleKey[] getClassAndFunctionNodes() {
+  public static Kind[] getClassAndFunctionNodes() {
     return Arrays.copyOf(CLASS_AND_FUNCTION, CLASS_AND_FUNCTION.length);
   }
 
-  public static GrammarRuleKey[] getControlStructureNodes() {
-    return Arrays.copyOf(CONTROL_STRUCTURE, CONTROL_STRUCTURE.length);
-  }
+//  public static Kind[] getControlStructureNodes() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)